beets/dbcore/queryparse.py

```diff
@@ -109,7 +113,10 @@ def construct_query_part(model_cls, prefixes, query_part):
             return query_class(pattern)
 
     key = key.lower()
-    return query_class(key.lower(), pattern, key in model_cls._fields)
+    if query_class is query.NoneQuery:
+        return query_class(key.lower(), pattern)
```

Comment:
Any particular reason why `NoneQuery` doesn't get the `fast` parameter to its `__init__` method?